.github/workflows/lint-python.yml

```diff
@@ -6,14 +6,14 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
-      - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
+      - run: pip install setuptools build wheel bandit black codespell flake8 isort mypy pytest pyupgrade safety
```

Comment:
Please keep dependencies sorted to make it easy to spot missing dependencies and difficult insert duplicates.

Given that build pip setuptools and wheel are used to build and install other dependencies, it is traditional to install that group separately before installing other deps.

Reply:
done - you might want to cancel outdated GHA runs 155-157